process-document/src/main/kotlin/com/ritense/processdocument/repository/ProcessDefinitionCaseDefinitionRepository.kt

```diff
@@ -42,4 +43,17 @@ interface ProcessDefinitionCaseDefinitionRepository:
         @Nullable @Param("startableByUser") startableByUser: Boolean?,
         @Nullable @Param("canInitializeDocument") canInitializeDocument: Boolean?
     ): List<ProcessDefinitionCaseDefinition>
+
+    @Query(
+        """
+    SELECT pdcd
+    FROM ProcessDefinitionCaseDefinition pdcd
+    WHERE pdcd.id.caseDefinitionId = :caseDefinitionId
+      AND pdcd.id.processDefinitionId.id = :processDefinitionId
+    """
+    )
+    fun findByCaseDefinitionIdAndProcessDefinitionId(
```

Comment:
The whole query might not be needed if you rename the function to:
`findAllByIdCaseDefinitionIdAndIdProcessDefinitionIdId(`